<!-- 
   <table border="1" cellspacing="5" cellpadding="2" align="MBCF">
        <tr>
            <td><a href="http://mbcf220.dfci.harvard.edu/"><img alt="DFCI" src="http://mbcf149.dfci.harvard.edu/images/logotop640.jpg" border="0"></a></td>
        </tr>
    </table>
<tr>
<br>
</tr>

<i>page last updated 02/03/13</i><P>
<a name="MiSeq" id="MiSeq"></a><strong><font color="#0000FF"><font size="+2"> MBCF Genomics Core</font></font></strong><br>
<br>
</b>
-->
<strong><font color="#0000FF"><font size="+2"> Illumina MiSeq Service</font></font></strong><br>








<p>
<p> 
The Illumina MiSeq is a Next-Gen DNA sequencer with a single-laned flowcell that is capable of generating 15 million single end or 30 million paired-end reads.
Illumina prepared libraries are loaded directly onto the MiSeq where amplified clusters are generated on the flowcell and sequencing by synthesis occurs.
There are 3 different sized reagent kits available, 50cycles, 300cycles, and 500cycles. In addition, the MiSeq can sequence 2 indices that are 8bp in length.<br>
<p>
<font size="+1">Common Run Programs:</font>
<ul>
<pre>
<li>1 X 50bp  (SE50),  600-800 Mb </li>
<li>2 X 30bp  (PE30),  720-960 Mb </li>
<li>2 X 150bp (PE150), 3.6-4.8 Gb </li>
<li>2 X 250bp (PE250), 6.0-8.0 Gb </li>
</pre>
</ul>
</p>
<strong><font color="#0000FF"><font size="+2">MiSeq Workflow</font></font></strong><br>
<br>
<ol><font size="+1">
<pre>
<li>Register an iLabs account </li>
<li>Consultation </li>
<li>Build Library </li>
<li>Submit an iLabs request</li>
<li>Drop off sample</li>
<li>Sequencing</li>
<li>Transfer Data </li>
<li>Analyze Data</li>
</font></font>
</pre>
</ol>

You can register and talk to us on Monday, build a library on our IntegenX robot on Tuesday, drop off the sample on Wednesday and by Wednesday night load your Bam file into a viewer that night. From then on, all your days can be Wednesdays. Drop off sample, look at your genome, same day.<P>

<strong><font color="#0000FF"><font size="+1">1. Register an iLabs Account at DFCI</font></font></strong><br>
If you are considering a running a sample or project on the MiSeq, you should first <a href="https://dfci.ilabsolutions.com/account/signup/1?sc_id=7607"target="_blank"> open an iLabs account</a>. You will receive a "welcome" email within one business day after which you can <a href="https://dfci.ilabsolutions.com/account/login"target="_blank"> login to iLabs </a>, navigate to MBCF Genomics Core and place a service request. Once you are logged in you can view all of the Molecular Biology Core Facilities most current core services, pricing, and place requests<br>
<br>
<strong><font color="#0000FF"><font size="+1">2. Consultation</font></font></strong><br>
If you are new to NGS,  new to MiSeq, or have a new project it is a good idea to <a href="mailto:zherbert@research.dfci.harvard.edu?cc=paul_morrison@dfci.harvard.edu&subject=MiSeq Sequencing"><font size="+1">Contact Us</a></font> to briefly discuss your objectives and address any potential challenges.
<br>
<br>
<strong><font color="#0000FF"><font size="+1">3. Build a Library</font></font></strong><br>
A Miseq library should consist of DNA fragments or amplicons that are between 200-800bp with Illumina PE adapter sequences. For optimal clustering and basecalling it helps to have a relatively narrow size range (~200bp) "Low Diversity" libraries with homogenous base composition --an extreme example would be a single amplico -- are problematic and can require phiX control library spike-in of 10%-40%, or an alternative library construction approach for successful clustering and sequencing. If you are concerned about library diversity this should be addressed during the consultation.<br>
<br>
<b>Important Note:</b> Even if you are only running single-end sequencing, the MiSeq requires PE adapters for cluster generation. Libraries prepared using older HiSeq or GAII SE kits may not compatible with the MiSeq.<br>
<br>
<strong><font color="#0000FF"><u>Nextera Library Prep Service</u></font></strong><br>
Nextera is a fast library prep method that requires only a small amount of DNA -as little as 1ng of input. This method uses a transposase to simultaneously fragment and tag DNA with Illumina adapters ("Tagmentation) followed by a limited cycle PCR reaction to incorporate sequencing primers and indices.<br>
<br>	
Nextera: 50ng input DNA -larger yield, 5 PCR cycles<br>
<br>
NexteraXT: 1ng input DNA -smaller yield (ideal for multiplexing), 12 PCR cycles.<br>
<br>
<strong><font color="#0000FF"><u>TruSeq Library Prep Service</u></font></strong><br>
This is the traditional Illumina sample prep method that includes end repair for sheared DNA, adapter ligation, size selection and amplification. We currently do not offer this as a service. However, we will be testing an Apollo 324 from IntegenX from January to March 2013 for automated TruSeq library preparation. Please contact us if you are tired of building your own TruSeq libraries and would like a robot to do it for you. (There is no obligation to run your sample on our MiSeq either if you just want to take your library and run)<br>
<br>	
<strong><font color="#0000FF"><font size="+1">4. Submit an iLabs Request</font></font></strong><br>
Log into your iLabs account and select one of our MiSeq run options. Fill in the request form to the best of your ability and in as much detail as possible. You can still submit your request even if you are unsure about some of the information. You also have the ability to upload files such as spreadsheets and bioanalyzer trace pdfs to the request. Providing lots of information about your sample(s) will help maximize the quality and quantity of data output.
<br>
<br>
<strong><font color="#0000FF"><font size="+1">5. Sample Drop-off</font></font></strong><br>
<b>We would like at least 5-10ul of DNA library at a 10nM concentration (more is always better...contact us if less is the case).</b>
Samples can be dropped off in the MBCF Fish room near 3rd floor elevator lobby of Smith, CLS building drop off on the 11th floor, directly to our lab on the 4th floor of DFCI Fenway Campus, or shipped <a href="http://mbcf149.dfci.harvard.edu/docs/contactinfo.html" target="_blank">via FedEx</a>. <br>
<br>
<strong><font color="#0000FF"><font size="+1">6. Sequencing</font></font></strong><br>
Actual run times vary from 4hrs to 40hrs depending on the number of cycles requested. Our turnaround time is usually within 2-3 days of receiving the sample and sometimes as fast as same day possible for shorter runs.<br>
<br>
<strong><font color="#0000FF"><font size="+1">7. Data Transfer</font></font></strong><br>
When the MiSeq run is complete data will be made available on our web server along with access instructions. Data and run metrics can also be made available on Illuminaâ€™s BaseSpace cloud environment.<br>
<br>
<strong><font color="#0000FF"><font size="+1">8. Analysis</font></font></strong><br>
The MBCF will provide compressed raw fastq data files (.fastq.gz). We can also align reads to a reference using a number of different softwares to provide bam formatted files (.bam and .bai). The analysis strategy should be discussed during the consultation.<br>
Here are some analysis resources:
<ul>
<li> <a href="https://main.g2.bx.psu.edu/root"target="_blank">Galaxy</a> Free public server with an array of NGS analysis tools.
<li><a href="http://www.clcbio.com/" target="_blank">CLCbio Genomics Workbench </a>(desktop/server based analysis)<br> if you're inside DFCI contact research computing for access to our server license.</li>
<li> <a href="http://basespace.illumina.com/" target="_blank"> Illumina BaseSpace</a> (data transfer/cloud storage)<br>
<li><a href="http://www.broadinstitute.org/igv/" target="_blank"> Broad Integrative Genomics Viewer (IGV)</a> (Viewer)</li>
<li><a href="http://genome.ucsc.edu/" target="_blank"> UCSC Genome Browser </a> (Viewer)</li>
</ul>
<p>
<strong><font size="+1">Instrumentation</font></font></strong><br>
<ul>
<li>3 MiSeqs (Illumina) -"Next-Gen" DNA Sequencers </li>
<p>
<a href="http://mbcf149.dfci.harvard.edu/htdocs/MiSeq/MiSeq.jpg"><img width="200" height="160" src="http://mbcf149.dfci.harvard.edu/MiSeq/MiSeq.jpg"> </a><a href="http://mbcf149.dfci.harvard.edu/htdocs/MiSeq/MiSeq.jpg"><img width="200" height="160" src="http://mbcf149.dfci.harvard.edu/MiSeq/MiSeq.jpg"> </a><a href="http://mbcf149.dfci.harvard.edu/htdocs/MiSeq/MiSeq.jpg"><img width="200" height="160" src="http://mbcf149.dfci.harvard.edu/MiSeq/MiSeq.jpg"> </a><p> 


<li>TapeStation (Agilent) -fibrary Fragment Size QC </li>
<a href="http://mbcf149.dfci.harvard.edu/htdocs/MiSeq/TapeStation.jpg"><img width="200" height="160" src="http://mbcf149.dfci.harvard.edu/MiSeq/TapeStation.jpg"> </a>
<li>Pippin Prep (Sage Science) -automated agarose gel size selection </li>
<a href="http://mbcf149.dfci.harvard.edu/htdocs/MiSeq/Pippin.jpg"><img width="200" height="160" src="http://mbcf149.dfci.harvard.edu/MiSeq/Pippin.jpg"> </a>


<li>Qubit 2.0 Fluorometer (Life Tech) -DNA mass determination </li>
<img width="200" height="160" src="http://mbcf149.dfci.harvard.edu/MiSeq/qubit2.jpg"> <P>


<li>Eco qPCR (Illumina) library quantification </li>
<img width="200" height="160" src="http://mbcf149.dfci.harvard.edu/MiSeq/ecoqpcr.jpg">  <P>


<li><a href="http://integenx.com/wp-content/uploads/2012/10/Apollo-324-Brochure-D006274-Rev-B.pdf"target="_blank">Apollo324 (IntegenX)</a> -automated TruSeq library prep </li>
</ul>
</b>
<strong><font color="#0000FF"><font size="+1">Education and Troubleshooting</font></font></strong><br>
<ul>
<li> Application notes for <a href="http://mbcf149.dfci.harvard.edu/miseq/appnote_miseq_denovo.pdf" target="_blank">Small genome denovo</a>, <a href="http://mbcf149.dfci.harvard.edu/miseq/appnote_miseq_ecoli.pdf" target="_blank">e. coli</a>, <a href="http://mbcf149.dfci.harvard.edu/miseq/appnote_miseq_ffpe.pdf" target="_blank">FFPE</a>, and <a href="http://mbcf149.dfci.harvard.edu/miseq/appnote_miseq_16S.pdf" target="_blank">16S</a>
<li> <a href="https://illumina.coursehost.com" target="_blank">Illumina Training</a> Here are some very basic online training videos about the technology and sample prep. You will need to set up an account.

<li><a href="http://seqanswers.com/forums/index.php" target="_blank">SEQanswers </a><br> Online forum that covers all matters DNA sequencing from service providers and sample prep to applications and informatics. Ask the experts and see what other people are asking.</li>
<li><a href="http://www.illumina.com/support/sequencing/sequencing_instruments/miseq/questions.ilmn" target="_blank">Illumina MiSeq FAQ </a><br></li>
<li><a href="http://www.abrf.org/index.cfm/list.index" target="_blank">ABRF Discussion Forum</a> <br>
Covers a broad range of biomolecular topics including DNA sequencing and biocomputing.
</li>


<p>
<strong><font color="#0000FF"><font size="+1">Library Prep Kits; Ordering and More Info</font></font></strong><br>
<a href="http://www.illumina.com/systems/miseq/kits.ilmn" target="_blank"> http://www.illumina.com/systems/miseq/kits.ilmn </a>
<br>



</ul>

<i>page last updated 02/03/13</i>
